README.md

```diff
@@ -56,3 +59,6 @@ the Arrow-specific specifications (Arrow layout and extension type metadata).
 * [geoarrow-rs](https://github.com/geoarrow/geoarrow-rs/): Rust implementation of the
   GeoArrow specification and bindings to GeoRust algorithms for efficient spatial
   operations on GeoArrow memory. Includes JavaScript (WebAssembly) bindings.
+* [geoarrow-python](https://github.com/geoarrow/geoarrow-python): Bindings to geoarrow-c
+  and geoarrow-rs that provide integrations with libraries like pyarrow, pandas, and
+  geopandas.
```

Comment:
Maybe it makes sense to add `geoarrow-wasm` to this list as well? https://www.npmjs.com/package/geoarrow-wasm or https://github.com/geoarrow/geoarrow-rs/blob/main/js/README.md are fine links